pkg/virt-controller/watch/volume-migration/volume-migration.go

```diff
@@ -359,10 +360,15 @@ func ValidateVolumesUpdateMigration(vmi *virtv1.VirtualMachineInstance, vm *virt
 	for _, v := range migVolsInfo {
 		volMigMap[v.VolumeName] = true
 	}
+	persistBackendVolName := backendstorage.CurrentPVCName(vmi)
 	for _, v := range vmi.Status.VolumeStatus {
 		if v.PersistentVolumeClaimInfo == nil {
 			continue
 		}
+		// Skip the check for the persistent VM state, this is handled differently then the other PVCs
+		if v.Name == persistBackendVolName {
```

Comment:
So the only problem was the validation bit? How does the TPM PVC get properly migrated?

Reply:
Well, if it is RWX then we don't need to do anything, otherwise, it should be added to disk to be copied as it is the case for container disks.

Reply:
Ah, and that part is already covered I guess?

Reply:
I mean, if a user asks for the backend state PVC to be migrated, that works? even though it's not in the VM spec?

Reply:
no, it doesn't work. The backend volume doesn't appear in the list of the volume for the VM, it needs to be handled separatly